safe_eth/eth/account_abstraction/bundler_client.py

```diff
@@ -195,4 +208,12 @@ def supported_entry_points(self) -> List[ChecksumAddress]:
             "params": [],
             "id": 1,
         }
-        return self._do_request(payload)
+        result = self._do_request(payload)
+        if result and isinstance(result, list):
+            return [
+                ChecksumAddress(HexAddress(HexStr(x)))
+                for x in result
+                if isinstance(x, str)
+            ]
+        else:
```

Comment:
We don't need the else, right? I mean is the default return.

Reply:
We have discussed this previously, and we commented on maintaining it

Reply:
Some tools recommend to remove the `else` after a `return` (like pylint if I'm not wrong). I would remove it. Also, please don't use `x` for a list comprehension, and use the name of the variable you expect, maybe `address` in this case

Reply:
Ok, makes sense, updated [here](https://github.com/safe-global/safe-eth-py/pull/1308/commits/d5124b9ba7afdab46bf2237d16f63c7072d97114).